### DISCUSSION TITLE
Slides appear only on the left

### DISCUSSION THREAD

**Discussion by tabtape** [nolimits4web/swiper#7038] [General]
Hi everyone.
Does anyone know why this happens?
There are three slides but only two appear and on the left. The third appears only when I click on the right arrow. If I keep clicking, the images scroll but then an empty slide always appears.

![slide 1](https://github.com/nolimits4web/swiper/assets/145197714/1affef6d-0669-4549-9739-9299cad160df)
![slide 3](https://github.com/nolimits4web/swiper/assets/145197714/ad262b1a-6e2d-409e-8901-c5fa7ad42845)

In the code there is the _transform:translate3d_ property which seems to have automatically moved everything to the left. It's a property that I can't change. If I apply the "important!" everything stops.

I just added the images and set the javascript... did I make a mistake there? but I don't understand what would need to be set to get a working slider.

How can I solve it?

There is already a discussion about this problem ( https://github.com/nolimits4web/Swiper/issues/2318#event-1338372815 ) but the solution is not clear, or rather it has not really been reported.

Thank you.

**Comment by yxarata** [comment]
Bonjour, 
J'ai le même problème https://youtu.be/vi5AScIOB1o
sur Swiper 12

J'ai testé en ajoutant "normalizeSlideIndex: false," comme dans la discution #5075 
mais ça ne semble rien changer

Ci-dessous le code JS

`var swiper = new Swiper(".swiper-portfolio", {
        autoplay: {
            delay: 2500,
            disableOnInteraction: true,
        },
        slidesPerView: 6,
        spaceBetween: 30,
        centeredSlides: true,
        normalizeSlideIndex: false,
        loop: true,
        navigation: {
            nextEl: ".swiper-portfolio-next",
            prevEl: ".swiper-portfolio-prev",
        },
        breakpoints: {
            400: {
                slidesPerView: 2,
            },
            600: {
                slidesPerView: 3,
            },
            768: {
                slidesPerView: 4,
            },
            992: {
                slidesPerView: 5,
            },
            1365: {
                slidesPerView: 6,
            }
        }
    });`
